book-exchange-website/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "book-exchange-website",
```

Comment:
please rename this as mentioned above.

Reply:
I renamed it to `alexandria-ui` as another commit.